docs/ug/sharingReports.md

```diff
@@ -29,10 +29,10 @@ You can point the recipients to the [_**Using reports**_](usingReports.html) sec
 
 As RepoSense reports are in a web page format, you can publish a report by simply uploading it onto any web hosting service. Given below are several options that not only allow to publish reports, but **also enable various levels of automating** the entire process (e.g., automatically update the report daily).
 
+Published reports can additionally be embedded in other websites through `iframes`. Simply click the clipboard icon to generate and copy the iframe for your desired section of the report. Paste the iframe in any HTML supported document to render it.
```

Comment:
May I suggest adding a separate section for this to [Sharing Reports](https://reposense.org/ug/sharingReports.html) and including an example for each use case in the new section?